src/targets/browser/launcher.ts

```diff
@@ -134,7 +134,10 @@ export async function launch(
       const endpoint = await waitForWSEndpoint(browserProcess, cancellationToken);
       transport = await WebSocketTransport.create(endpoint, cancellationToken);
     } else {
-      const endpoint = await waitForDebuggerServerOnPort(suggestedPort, cancellationToken);
+      const endpoint = await retryGetWSEndpoint(
+        `http://localhost:${suggestedPort}`,
+        cancellationToken,
```

Comment:
So this is canceled after the timeout? Where does that happen?

Reply:
It is `_boot` in the Binder creates a CancellationTokenSource for launch and attach requests, on which cancellation is requested after a certain amount of time.